andela/caret-bn-backend
# #168781713 - Users can view travelled Destinations

#### What does this PR do?

This PR enables authenticated users to view the top 5 most travelled locations in the barefoot nomad system. 

#### Description of Task to be completed?

- Allow users and managers to view the top 5 travelled locations in the barefoot nomad system.

#### How should this be manually tested?

1. Clone this repository
2. Run ` sequelize db:migrate `
3. Run ` sequelize db:seed:All `
4. Run ` npm run dev `
5. Log into the application at `api/v1/users/login`
6. Using an  authenticated user send a GET request to `/api/v1/destinations/top-locations`
7. It should display the top 5 locations visited in barefoot nomad. 
8. Incase 4 locations are displayed. Manually change the status of one or more requests to status id `3` and run the endpoint again

#### Any background context you want to provide?

N / A


#### What are the relevant pivotal tracker stories?

###### #168781713

#### Screenshots (if appropriate)

<img width="1228" alt="Screen Shot 2019-11-05 at 08 18 23" src="https://user-images.githubusercontent.com/13886438/68183365-e4366400-ffa4-11e9-9738-c1834be9116f.png">
<img width="1264" alt="Screen Shot 2019-11-05 at 08 18 30" src="https://user-images.githubusercontent.com/13886438/68183366-e4366400-ffa4-11e9-9972-20c7a22e7ff2.png">
<img width="818" alt="Screen Shot 2019-11-05 at 08 18 35" src="https://user-images.githubusercontent.com/13886438/68183367-e4366400-ffa4-11e9-8925-d420bd37b3e3.png">


#### Questions:
N / A
